redisinsight/ui/src/packages/ri-explain/src/Node.tsx

```diff
@@ -99,9 +100,9 @@ export function ProfileNode(props: INodeProps) {
     <div className="ProfileContainer" id={`node-${id}`}>
       <div className="Main">
         <div className="InfoData">
-          <EuiToolTip delay="long" content={infoData}>
+          <RiTooltip delay={500} content={infoData}>
```

Comment:
I see this magic number on a few places and I wonder whether the new component comes with some predefined delays, like EUI?

Otherwise, we may put this magic number behind a shared constant (something like `TOOLTIP_DELAY_LONG=500`) so we can easily update it in the future, if necessary.